test/e2e/framework/timeouts.go

```diff
@@ -58,6 +59,9 @@ type TimeoutContext struct {
 	// PodList is how long to wait for the pod to be listable.
 	PodList time.Duration
 
+	// PodEventTimeout is how much we wait for a pod event to occur.
+	PodEvent time.Duration
```

Comment:
interesting, we should not have any test depending on this https://github.com/kubernetes/kubernetes/issues/71646